mdtraj/core/topology.py

```diff
@@ -636,15 +636,15 @@ def __eq__(self, other):
 
         return True
 
-    def add_chain(self):
+    def add_chain(self, chain_id = None):
```

Comment:
[pep8](https://www.python.org/dev/peps/pep-0008/) suggestion
```suggestion
    def add_chain(self, chain_id=None):
```

Reply:
Done.